Templates.xcodeproj/project.pbxproj

```diff
@@ -216,7 +216,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "if [[ ! $CI ]]; then\n    rake lint:tests\n    rake lint:output\nfi";
+			shellScript = "if [[ ! $CI ]]; then\n  [[ -r ~/.profile ]] && source ~/.profile\n  [[ `which rbenv` ]] && eval \"$(rbenv init - --no-rehash sh)\"\n  if [[ `which bundler` ]]; then\n    bundle exec rake lint:tests\n    bundle exec rake lint:output\n  else\n    echo \"warning: bundler not found, can't lint code. Please install bundler then run bundle install.\"\n  fi\nfi";
```

Comment:
Right, but I don't use a `.profile` anymore. I have zprezto, which loads `rbenv` automatically. No idea what happens if I try to load `rbenv` twice.